dojoup/dojoup

```diff
@@ -133,10 +136,21 @@ main() {
     RELEASE_URL="https://github.com/${DOJOUP_REPO}/releases/download/${DOJOUP_TAG}/"
     BIN_ARCHIVE_URL="${RELEASE_URL}dojo_${DOJOUP_VERSION}_${PLATFORM}_${ARCHITECTURE}.$EXT"
 
+    # Check if the version mentioned by user exists in the Dojo repository.
+    if ! curl --output /dev/null --silent --head --fail "$BIN_ARCHIVE_URL"; then
+      say "Version ${DOJOUP_VERSION} does not exist at ${RELEASE_URL%/}"
+      say "Please mention a valid version."
```

Comment:
We may say to the user that if `-v` is not used, the latest stable version is automatically installed, wdyt?

Reply:
Changed it to this message: 

`Please specify a valid version, or omit -v to install the latest stable version automatically.`